app/views/transferAgreementAlreadyCompleted.scala.html

```diff
@@ -0,0 +1,104 @@
+                transferAgreementForm("publicRecord"),
+                Symbol("_label") -> "I confirm that the records are Public Records.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("crownCopyright"),
+                Symbol("_label") -> "I confirm that the records are all Crown Copyright.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("english"),
+                Symbol("_label") -> "I confirm that the records are all in English.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            <!-- DRO confirmation -->
+            @inputSingleCheckbox(
+                transferAgreementForm("droAppraisalSelection"),
+                Symbol("_label") -> "I confirm that the Departmental Records Officer (DRO) has signed off on the appraisal and selection decision.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("droSensitivity"),
+                Symbol("_label") -> "I confirm that the Departmental Records Officer (DRO) has signed off on the sensitivity review.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+            @inputSingleCheckbox(
+                transferAgreementForm("openRecords"),
+                Symbol("_label") -> "I confirm that all records are open and no Freedom of Information (FOI) exemptions apply to these records.",
+                Symbol("_value") -> "true",
+                Symbol("_smallCheckbox") -> true,
+                Symbol("_checkedOption") -> "true",
+                Symbol("_requiredOption") -> true,
+                Symbol("_disabledOption") -> "true"
+            )
+        </div>
+        <!-- Buttons -->
+        <div class="govuk-button-group">
+            <!-- Continue -->
```

Comment:
I'd get rid of these comments - they code they explain is already clear enough. 🙂